index.js

```diff
@@ -217,15 +230,19 @@ function adapter(uri, opts){
 
     debug('removing %s from rooms %s', id, Object.keys(rooms));
 
-    Adapter.prototype.delAll.call(this, id);
```

Comment:
the delAll command was handling the code you've written below. 

See:

https://github.com/socketio/socket.io-adapter/blob/master/index.js#L80-L93